dev/EnvironmentManager/ChangeTracker/EnvironmentVariableChangeTrackerHelper.h

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+#pragma once
+
+#include <windows.h>
+#include <memory>
+#include <wil/result.h>
+#include <appmodel.h>
+#include <winrt/Windows.Management.Deployment.h>
+#include <Microsoft.Windows.System.EnvironmentManager.h>
+#include <AppModel.Identity.h>
+#include <Security.IntegrityLevel.h>
+#include <Microsoft.Windows.System.EnvironmentManager.Insights.h>
+#include <WindowsAppRuntime.SelfContained.h>
+
+
+using namespace winrt::Microsoft::Windows::System::implementation;
+
+static bool DoesChangeTrackingKeyExist()
+{
+    const auto subKey = std::filesystem::path{ LR"(Software\Microsoft\AppModel\RegistryWriteVirtualization\ExcludedKeys\HKEY_CURRENT_USER/Software/ChangeTracker)" };
+    wil::unique_hkey keyToChangeTracker{};
+    LSTATUS openResult{ RegOpenKeyEx(HKEY_LOCAL_MACHINE, subKey.c_str(), 0, KEY_READ | KEY_WOW64_64KEY, keyToChangeTracker.put()) };
+
+    if (FAILED_WIN32(openResult))
+    {
+        EnvironmentManagerInsights::TraceLoggingInfo("Could not find the change tracking HKLM key.  Changes will not be tracked");
```

Comment:
Where's this log to? If telemetry then the english prose shouldn't be necessary. If the eventlog where, what about localization, yadda yadda. Elsewhere?

Reply:
True.  Since a LOG_IF_WIN32 now exists.  The LOG_ will work.

This would go to telemetry and telemetry would not need a "HKLM key not found"